src/dream-app/index.ts

```diff
@@ -140,6 +141,12 @@ export default class DreamApp {
         this.encryption.columns.current.algorithm
       )
 
+    if (
+      !DreamAppAllowedPackageManagersEnumValues.includes(this.packageManager) &&
+      !EnvInternal.boolean('DREAM_BYPASS_PACKAGE_MANAGER_CHECK')
```

Comment:
We've been moving away from environment variables to control this kind of thing. Would this be better as an option that was passed?